ethpm_cli/manager.py

```diff
@@ -0,0 +1,87 @@
+from ethpm.typing import Manifest
+from eth_utils import to_text
+from ethpm.backends.ipfs import BaseIPFSBackend
+from ethpm.utils.ipfs import is_ipfs_uri
+import json
+from pathlib import Path
+from typing import Dict
+
+from eth_utils.toolz import assoc
+from ethpm.typing import URI
+
+from ethpm_cli.exceptions import InstallError
+from ethpm_cli.package import Package
+from ethpm_cli._utils.ipfs import get_ipfs_backend
+
+
+class Manager:
```

Comment:
Gotcha - I ended up moving some functions to this class - since it was easier to handle the ipfs backend by setting it as an attribute to the `Manager` class. What are your thoughts on setting the ipfs backend in an environment variable? It might be simpler than passing it back/forth between the `Manager` and `Package` classes? And could simplify some of the code

Reply:
Ahh nvm this - just saw the `Config` comment

Reply:
Sounds like you found a way forward on this, however, I am almost always :+1: on environment variable based configuration.